sys/kern/kenv.c

```diff
@@ -1,7 +1,7 @@
 #include <sys/kenv.h>
 #include <sys/libkern.h>
 
-static char **_kenvp;
+static char **_kenvp = (char *[1]){NULL};
```

Comment:
Short comment why it is needed?

Reply:
It was needed because of `_kenvp` dereference in `init_klog`, but ss I'm thinking about it now, the aforementioned dereference shouldn't be present. I have changed the code so that
1. klog is initialized with the default mask.
2. `klog_update_mask` set the appropriate mask (if supplied).
Now the pointed change is obsolete.